lib/kitchen/driver/docker.rb

```diff
@@ -181,6 +204,11 @@ def build_dockerfile
           RUN mkdir -p /etc/sudoers.d
           RUN echo '#{username} ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers.d/#{username}
           RUN chmod 0440 /etc/sudoers.d/#{username}
+          RUN [ ! -d /home/kitchen/.ssh ] && mkdir /home/kitchen/.ssh
+          RUN chown -R kitchen:kitchen /home/kitchen/.ssh
+          RUN chmod 0700 /home/kitchen/.ssh
+          RUN echo '#{IO.read(config[:public_key])}' >> /home/kitchen/.ssh/authorized_keys
```

Comment:
Maybe we should use a variable, something along the lines of `public_key` or `public_key_contents`? Assign it prior to `base =`.